index.js

```diff
@@ -1,12 +1,11 @@
-function inject(bot) {
+module.exports = function inject(bot) {
     bot.autototem = {}
-    bot.autototem.totem = () => {
+    bot.autototem.equip = () => {
         if (bot.inventory.slots[45] != null) return
         const totem = bot.inventory.findInventoryItem('totem_of_undying', null, null)
-        if (totem) bot.equip(totem, 'off-hand')
+        if (totem) {
+            bot.inventory.requiresConfirmation = false
+            bot.equip(totem, 'off-hand')
+        }
     }
 }
-
-module.exports = {
```

Comment:
Please don't remove this and keep the `module.exports` the way it was, it does not work otherwise

Reply:
ok